Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -23,8 +23,7 @@ open class WayNameLabel: StylableLabel {
     
     // When the map style changes, update the sprite repository and the label.
     func updateStyle(styleURI: StyleURI?) {
-        guard let styleURI = styleURI else { return }
-        spriteRepository.updateStyle(styleURI: styleURI, representation: representation) { [weak self] in
+        spriteRepository.updateStyle(styleURI: styleURI) { [weak self] in
```

Comment:
Whenever we use shared instance of `SpriteRepository` let's access it via: `SpriteRepository.shared` to make it clear that it's a singleton.

Reply:
Yes, right now we only use it in `WayNameLabel` and `InstructionPresenter`, they both using the `SpriteRepository.shared` as a singleton.